docs/data-sources/infra_join_tokens.md

```diff
@@ -0,0 +1,48 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "bloxone_infra_join_tokens Data Source - terraform-provider-bloxone"
+subcategory: ""
+description: |-
```

Comment:
How did you generate this? We don't have a make target yet for this

Reply:
`go generate` builds it

Reply:
`go generate` builds it

Reply:
`go generate` builds it. It is also part of the checks in Github actions. It will fail if `go generate` is not run